src/app/users/getServices.js

```diff
@@ -13,43 +13,39 @@ const getOrganisations = async (userId, correlationId) => {
   }
 
   const userMap = [];
-  const organisations = [];
-  // orgServiceMapping.forEach((orgSvcMap) => {
-  for (let i = 0; i < orgServiceMapping.length; i += 1) {
-    const orgSvcMap = orgServiceMapping[i];
-    let org = organisations.find(o => o.id === orgSvcMap.organisation.id);
-    if (!org) {
-      org = {
-        id: orgSvcMap.organisation.id,
-        name: orgSvcMap.organisation.name,
-        services: [],
+
```

Comment:
might be worth checking this out - i think it was done as for loop as it was more performant/didnt do strange things that map did